ignite/contrib/handlers/lr_finder.py

```diff
@@ -336,9 +338,10 @@ def attach(
             self._detach(trainer)
             # restore to_save and reset trainer's state
             obj = torch.load(cache_filepath.as_posix())
-            trainer.state = None
+            trainer.load_state_dict(obj["trainer"])
```

Comment:
Can it conflict with others loads ?

Reply:
> Can it conflict with others loads ?

Which others loads ? 
I didn't include trainer's state dict into `to_save` in order to keep it immutable as Mapping requires...

Reply:
`lr_finder` just needs to restore state of trainer, that was a stupid remark !